Resources/Prototypes/Recipes/Reactions/medicine.yml

```diff
@@ -454,3 +454,14 @@
       amount: 1
   products:
     SpaceDrugs: 3
+    
+- type: reaction
+  id: Ultravasculine
+  reactants:
+    Histamine:
+      amount: 2
+    Plasma:
+      amount: 5
+      catalyst: true
+  products:
+    Ultravasculine: 3
```

Comment:
should produce 2 imo

Reply:
I think potency needs to be fixed first since plants can't gain potency yet